aiven_mysql_migrate/main.py

```diff
@@ -1,27 +1,49 @@
 # Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
 from aiven_mysql_migrate import config
 from aiven_mysql_migrate.migration import MySQLMigrateMethod, MySQLMigration
+from dataclasses import asdict
 
+import json
 import logging
 
 LOGGER = logging.getLogger(__name__)
 
+_LOG_LEVELS = {
+    "debug": logging.DEBUG,
+    "info": logging.INFO,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+}
 
-def setup_logging(*, debug=False):
+
+def setup_logging(*, log_level: str = "info"):
     log_format = "%(asctime)s\t%(name)s\t%(levelname)s\t%(message)s"
-    log_level = logging.DEBUG if debug else logging.INFO
+    log_level = _LOG_LEVELS.get(log_level.lower(), logging.INFO)
     logging.basicConfig(level=log_level, format=log_format)
 
 
 def main(args=None, *, app="mysql_migrate"):
     """Migrate MySQL database from source to target, take configuration from CONFIG"""
     import argparse
     parser = argparse.ArgumentParser(description="MySQL migration tool.", prog=app)
-    parser.add_argument("-d", "--debug", action="store_true", help="Enable debug logging.")
+    parser.add_argument(
+        "-d", "--debug", action="store_true", help="Enable debug logging. Deprecated: please use --log-level"
```

Comment:
I think we can just remove `debug` option, we don't use it and I doubt if someone else does.

Reply:
Thanks, will do. I was just unsure if someone is using it, so decided to support both the formats for the time being.